diff --git b/assets/www/SQLitePlugin.js a/assets/www/SQLitePlugin.js
index b838727..ce4d464 100755
--- b/assets/www/SQLitePlugin.js
+++ a/assets/www/SQLitePlugin.js
@@ -47,6 +47,8 @@
   pcb = function() {
     return 1;
   };
+
+  // XXX TBD fix callback(s):
   SQLitePlugin.prototype.executeSql = function(statement, params, success, error) {
     console.log("SQLitePlugin::executeSql[Statement]");
     pcb = success;
@@ -54,6 +56,7 @@
       return 1;
     }), error, "SQLitePlugin", "executePragmaStatement", [this.dbname, statement, params]);
   };
+  // DEPRECATED AND WILL BE REMOVED:
   SQLitePlugin.prototype.executePragmaStatement = function(statement, success, error) {
     console.log("SQLitePlugin::executePragmaStatement");
     pcb = success;
@@ -61,6 +64,7 @@
       return 1;
     }), error, "SQLitePlugin", "executePragmaStatement", [this.dbname, statement]);
   };
+
   SQLitePluginCallback = {
     p1: function(id, result) {
       var mycb;
@@ -252,6 +256,10 @@
     cordova.exec(null, null, "SQLitePlugin", "executeSqlBatch", [this.dbname, transaction_queue[this.trans_id]]);
   };
   SQLiteFactory = {
+    // NOTE: this function should NOT be translated from Javascript
+    // back to CoffeeScript by js2coffee.
+    // If this function is edited in Javascript then someone will
+    // have to translate it back to CoffeeScript by hand.
     opendb: function() {
       var errorcb, first, okcb, openargs;
       if (arguments.length < 1) {
