diff --git b/assets/www/SQLitePlugin.js a/assets/www/SQLitePlugin.js
index c312905..ebcc9d7 100755
--- b/assets/www/SQLitePlugin.js
+++ a/assets/www/SQLitePlugin.js
@@ -55,6 +55,8 @@
   pcb = function() {
     return 1;
   };
+
+  // XXX TBD fix callback(s):
   SQLitePlugin.prototype.executeSql = function(statement, params, success, error) {
     console.log("SQLitePlugin::executeSql[Statement]");
     pcb = success;
@@ -62,6 +64,7 @@
       return 1;
     }), error, "SQLitePlugin", "executePragmaStatement", [this.dbname, statement, params]);
   };
+  // DEPRECATED AND WILL BE REMOVED:
   SQLitePlugin.prototype.executePragmaStatement = function(statement, success, error) {
     console.log("SQLitePlugin::executePragmaStatement");
     pcb = success;
@@ -69,6 +72,7 @@
       return 1;
     }), error, "SQLitePlugin", "executePragmaStatement", [this.dbname, statement]);
   };
+
   SQLitePluginCallback = {
     p1: function(id, result) {
       var mycb;
@@ -280,6 +284,10 @@
     this.run();
   };
   SQLiteFactory = {
+    // NOTE: this function should NOT be translated from Javascript
+    // back to CoffeeScript by js2coffee.
+    // If this function is edited in Javascript then someone will
+    // have to translate it back to CoffeeScript by hand.
     opendb: function() {
       var errorcb, first, okcb, openargs;
       if (arguments.length < 1) {
